plugins/kotlin/idea/tests/testData/inspectionsLocal/logging/placeholderCountMatchesArgumentCount/log4j/debug.kt

```diff
@@ -0,0 +1,12 @@
+// FIX: none
```

Comment:
Please, use `PROBLEM` directive to test a message as well.
For example:
```
// PROBLEM: Fewer arguments provided '2' than placeholders specified '3'
```

Reply:
Added for all cases `PROBLEM`. Thank you!